MAVProxy/modules/mavproxy_console.py

```diff
@@ -75,6 +77,16 @@ def __init__(self, mpstate):
             self.vehicle_menu = MPMenuSubMenu('Vehicle', items=[])
             self.add_menu(self.vehicle_menu)
 
+    def cmd_add(self, args):
+        new_field = []
+        new_field.append(args[0])
```

Comment:
isn't this the same as
  new_field = args[:]
?

Reply:
Yes